src/theme/foundations/textStyles.ts

```diff
@@ -1,51 +1,95 @@
 export const textStyles = {
   h1: {
-    fontFamily: '"JetBrains Mono", monospace',
+    fontFamily: 'heading',
     fontWeight: 700,
     fontSize: '2.75rem',
     lineHeight: '3.375rem',
     letterSpacing: '5%',
     color: 'body'
   },
   h2: {
-    fontFamily: '"JetBrains Mono", monospace',
+    fontFamily: 'heading',
     fontWeight: 400,
     fontSize: '1.5rem',
     lineHeight: 'auto',
     letterSpacing: '4%',
     color: 'body'
   },
+  homepageH1: {
```

Comment:
I dont see this used anywhere, do we need this?

Reply:
We probably don't tbh, I had included it for completeness when pulling in from the design system

Reply:
Or we look to add this to the actual h1 being used on the homepage... but like I said, I had just included it for completeness, but that style specifically doesn't really apply to this PR

Reply:
Looked into this... HomeHero has an h1 that is being styled using style props, but it doesn't strictly match the design. It probably does make sense to just keep that one-off styling for the homepage within that component, instead of pulling it into this `textStyles` file. It's out of scope for this PR, so I'll remove this `homepageH1` entry here, and in a separate PR I can update the styling for the h1 within `HomeHero` to match the design.